js/brushes/waterWave.js

```diff
@@ -0,0 +1,30 @@
+/* global mouseIsPressed, random, colorMode, fill, stroke, sin, ellipse,
+ HSB, frameCount, mouseX, mouseY */
+ var start = 0
+ var inc = 0.005
+ var xoff = 0
+
+window.brushes.waterWave = {
+  name: 'water wave',
+  draw: function () {
+    if (mouseIsPressed && !window.dragging) {
+      colorMode(RGB)
+      stroke(30,random(10,100),(100,150),random(30,255))
+  noFill()
```

Comment:
this code works great, but the formatting is off. this line should be indented the same way that the line above it is. the same is true for the code below this line. try to clean up the indentation so that all the lines within the same block (meaning between the same `{` and `}`) are always indented the same, for example:
```js
function () {
  example()
  example()
  if (something) {
    example()
    example()
  } else {
    example()
    example()
  }  
}
```

Reply:
Hi Nick,
Thank you for the good feedback!
I changed the word 'var' to 'let', and cleaned up the indentation.
Thank you :)